An Untitled Mining Game/An Untitled Mining Game/objects/obj_struct_spawner/Step_0.gml

```diff
@@ -2,8 +2,10 @@ if (keyboard_check_pressed(ord("B"))) {
 	building = !building;
 }
 
-
-if (building) {
+if (!building) {
+	window_set_cursor(cr_default);
+} else {
+	window_set_cursor(cr_none);
```

Comment:
I still think this is a very clever solution! I'd never think of it in a million years.